### DISCUSSION TITLE
Cursorless flashcards

### DISCUSSION THREAD

**Discussion by RonWalker22** [cursorless-dev/cursorless#589] [Show and tell]
I'm building some [flashcards](https://app.mochi.cards/decks/46fb32f4-7183-4f39-a6c5-f882ce43fa93/wvGjGBPb/Voice-Code) to help me learned cursorless. The deck is mostly limited to what is in the [docs](https://github.com/cursorless-dev/cursorless-vscode/tree/main/docs/user). I will be updating the deck periodically and expanding outside the [docs](https://github.com/cursorless-dev/cursorless-vscode/tree/main/docs/user) as I get more adventurous. 

To use the flashcards first download [mochi](https://mochi.cards/) for desktop then sign up for a free account, finally go to the [flashcards](https://app.mochi.cards/decks/46fb32f4-7183-4f39-a6c5-f882ce43fa93/wvGjGBPb/Voice-Code) and click "clone deck".

**Comment by pokey** [comment]
Cool!  How do you like mochi btw?  How's the spaced repetition?

**Reply by RonWalker22** [reply]
I love it. I don’t have much experience with the spaced repetition mode. There’s another mode that I use almost exclusively called “cram”. It's just a personal study style preference of mine. But I assume it works well, as spaced repetition is the app’s primary mode. Also there’s settings you can change to customize the space/time between repetitions so that you are not stuck with the default algorithm.

**Comment by JeongJuhyeon** [comment]
@RonWalker22 Your deck looks a lot better than the deck I made! I was thinking of incorporating animated gifs but you've already gone and done it! Is it okay with you if I convert your deck to Anki and share it here? While Mochi looks nice (especially the flat MD structure), I think for a lot of us long-time Ankiers we'd rather not run multiple flashcard apps at the same time.

**Comment by RonWalker22** [comment]
@JeongJuhyeon Yes!

**Reply by JeongJuhyeon** [reply]
Thank you :)

**Comment by DonnerWolfBach** [comment]
@RonWalker22 very, very cool! Especially the gifs are really neat. I wanted to export it to integrate it into my note taking app logseq (which has a little flash card mode, enough for my needs). However, I did not manage to download the gifs. Do you happen to know a way?

@JeongJuhyeon Did you manage to create the anki deck? I assume https://ankiweb.net/shared/info/1204743582 is a different one
